source/index.html.md

```diff
@@ -651,15 +651,60 @@ This endpoint schedule payments for this subscription using the dates and amount
 
 This endpoint schedule payments for this subscription using the order and amounts specified in the body
 
-> **Example request**
+> **Example body**
 
 ```json
+[
+  {
+    "amount": 20,
+    "date": "2022-04-30"
```

Comment:
I wasn't talking about changing the `order` field in the request body. I was talking only about the response. Otherwise this API call is no different from the other one. 
Restore the `order` field here.